contrib/google.golang.org/grpc/grpc.go

```diff
@@ -37,19 +36,19 @@ func startSpanFromContext(
 }
 
 // finishWithError applies finish option and a tag with gRPC status code, disregarding OK, EOF and Canceled errors.
-func finishWithError(span ddtrace.Span, err error, noDebugStack bool) {
+func finishWithError(span ddtrace.Span, err error, cfg *config) {
 	if err == io.EOF || err == context.Canceled {
 		err = nil
 	}
 	errcode := status.Code(err)
-	if errcode == codes.Canceled || errcode == codes.OK {
+	if cfg.nonErrorCodes[errcode] {
```

Comment:
Let's not even include `OK` in the map, since it's clearly not an error, and have something like this:
```go
if cfg.nonErrorCodes[errcode] || errcode == codes.OK {
```
It will also simplify things everywhere else, what do you think?

Reply:
Done.

IMO `nonErrorCodes` should also include `codes.OK` as only then it's the exhaustive list of non-error codes. For now I did it the way you suggested, because I really don't have any strong opinion about this.